tests/config/data/logging_format_interpolation_style.py

```diff
@@ -0,0 +1,16 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
@Pierre-Sassoulas Can we exclude the copyright hook for this directory?

Reply:
Sure !